geoportailv3/static/js/draw/featurepopup.html

```diff
@@ -52,12 +52,20 @@
 </div>
 <div ng-show="ctrl.editingStyle">
   <div class="feature-popup-body">
-    <a href class="close" ng-click="ctrl.editingStyle = false">&times;</a>
-    <app-style-editing app-style-editing-feature="ctrl.feature">
+    <app-style-editing app-style-editing-feature="ctrl.feature"
```

Comment:
I know it's unrelated to your PR but I'd rename the directive to `app-styleediting`, for consistency reasons.